selector_families.yml

```diff
@@ -554,3 +554,24 @@ selector_families:
   1948510578179542068:
     family: evm    
     name: "bitcoin-testnet-bsquared-1"
+  3789623672476206327:
+    family: evm
+    name: "bitcoin-testnet-bitlayer-1"
+  5535534526963509396:
+    family: evm
+    name: "ethereum-testnet-sepolia-bob-1"
+  6827576821754315911:
+    family: evm
+    name: "ethereum-testnet-sepolia-lens-1"
+  1216300075444106652:
+    family: evm
+    name: "cosmos-testnet-sei"
```

Comment:
Isn't Sei an L1? So in this case it would be something likg "sei-testnet" since cosmos is only the SDK that was used but it doesn't settle on Cosmos, correct?

Reply:
@RensR @andrevmatos can you double check here, should a cosmos SDK chain name start with cosmos or by itself?

Reply:
I agree it's a bit confusing.
Based on this doc: https://smartcontract-it.atlassian.net/wiki/spaces/DEPLOY/pages/599491447/Network+naming+proposal I think it means the anchor blockchain rather than the protocol. Because we usually ignore the protocol, for example many chains use the OP stack but we don't add Optimism to the name, we only add Ethereum.

Reply:
Is it a true, independent L1?
IIANM, Cosmos used to be a family of "L1s", somewhat independent but sharing consensus and security with the Cosmos Hub chain.

I think this requires some discretion, but if SEI is an app chain on top of Cosmos Hub, even if in some aspects it could be seen as an L1, I think we could/should treat these chains as instances anchored in the Hub, therefore calling it `cosmos-testnet-sei`. @mateusz-sekara wdyt?

Reply:
I don't have a strong opinion on this, to be honest. @andrevmatos you are da man behind the naming strategy so I always rely on your opinion on that.

> IIANM, Cosmos used to be a family of "L1s", somewhat independent but sharing consensus and security with the Cosmos Hub chain.

Yeah, but we don't add "optimism" to any opstack-based blokchains as @mohamed-mehany mentioned, so I guess dropping this `cosmos` prefix is fine. BSC technically is forked Ethereum, but we don't keep `ethereum` prefix there as well.

However, as blockchains are very deliberate about nuking testnets when they introduce large changes I would consider adding testnet's name (as we do for other chains like `ethereum-testnet-sepolia`). Therefore `sei-testnet-atlantic` (or whichever the ocean they use there )

Whatever we agree here, please make sure that the repository's readme is updated and confusing parts are adjusted accordingly https://github.com/smartcontractkit/chain-selectors?tab=readme-ov-file#contributing

Reply:
Yes, I agree we don't add `ethereum` or `optimism` to the chains using them as tech, but my point is that, from my understanding, Cosmos SDK chains don't share just technology, but are actually anchored to the hub (in a consensus-level), and can only be to the hub (there are no independent instances of Cosmos SDK chains as bsc-to-ethereum or optimism-based L2s).

So, to my understanding, Cosmos SDK chains have enough to be named as a standalone family than the other examples.

Reply:
@andrevmatos could you clarify what you mean by "sharing consensus and security"? Are you suggesting that it relies on connecting to the Cosmos chain, or does it simply reuse libraries from the SDK? If it's the latter, I don't think this means the chain is anchored to Cosmos. It's similar to other chains that utilize another chain's SDK, like Optimism or Arbitrum Nitro. For both, we typically omit mentioning the SDK because adding the name of the chain usually implies that it settles on another layer. Overcomplicating the naming might diminish its value.

Reply:
@andrevmatos That’s an interesting point, and I agree—if it genuinely connects to Cosmos for tasks like consensus or validation, then it makes sense to reflect that in the name.

However, based on what I’ve found in the chain’s documentation (please feel free to correct me if I’m mistaken, as I’m not a Cosmos expert), it appears everything is handled on-chain. For example, the validator documentation (https://www.docs.sei.io/dev-validators/overview) suggests that all operations occur within the chain itself. Additionally, they claim instant finality (https://www.docs.sei.io/dev-advanced-concepts/differences-with-ethereum#finality), which, from my understanding, shouldn’t be theoretically possible if the chain depends on another network for consensus.

The only thing I found that connects to Cosmos is their use of IBC (https://www.docs.sei.io/dev-tutorials/ibc-protocol), which seems to be a cross-chain connectivity protocol allowing them to transfer data between chains, including Cosmos.

Reply:
Hmm, right. From the [Cosmos SDK overview](https://docs.cosmos.network/main/learn/intro/overview)

> The goal of the Cosmos SDK is to allow developers to easily create custom blockchains from scratch that can natively interoperate with other blockchains. We further this modular approach by allowing developers to plug and play with different consensus engines this can range from [CometBFT](https://cometbft.com/) or [Rollkit](https://rollkit.dev/).

So, I may have misunderstood, and indeed Cosmos SDK is more like just a set of libraries for building standalone chains, which happen to share some common facilities, like the IBC. So I concede, let's keep the standalone naming here. Thank you and sorry for the noise.